tests/unit/amo/components/TestCategoryHead.js

```diff
@@ -40,36 +41,26 @@ describe(__filename, () => {
     expect(root.find('title')).toHaveText(category.name);
   });
 
-  it('renders a "description" meta tag when category description is available', () => {
-    const description = 'some description for a category';
-    const category = { ...fakeCategory, description };
-    const root = render({ category });
-
-    expect(root.find('meta[name="description"]')).toHaveLength(1);
-    expect(root.find('meta[name="description"]')).toHaveProp(
-      'content',
-      description,
-    );
-  });
-
-  it('does not render a "description" meta tag when category description is null', () => {
-    const category = { ...fakeCategory, description: null };
+  it('renders a HeadLinks component', () => {
+    const category = { ...fakeCategory };
     const root = render({ category });
 
-    expect(root.find('meta[name="description"]')).toHaveLength(0);
+    expect(root.find(HeadLinks)).toHaveLength(1);
   });
 
-  it('does not render a "description" meta tag when category description is empty', () => {
-    const category = { ...fakeCategory, description: '' };
-    const root = render({ category });
-
-    expect(root.find('meta[name="description"]')).toHaveLength(0);
-  });
+  it('renders a HeadMetaTags component', () => {
+    const category = { ...fakeCategory, type: ADDON_TYPE_THEME };
```

Comment:
in some other tests I notice you show both theme and extension versions. Should you do something similar here?